web/worker/jobhandler.py

```diff
@@ -80,7 +81,7 @@ def send_job_started_notification(self, job_ID, title, image_URL):
         args = {
             Request.ARGS_JOB_ID: job_ID,
             Request.ARGS_IMAGE_URL: image_URL,
-            Request.ARGS_TITLE: title,
+            Request.ARGS_TITLE: json_encode(title)[1:-1],
```

Comment:
why [1:-1] and not [0:-1]?

Reply:
The encode function itself adds quotes at the begin and the end of the string. With [1:-1] we get rid of both of them, with [0:-1] we would only remove those at the end.